params/config.go

```diff
@@ -551,6 +533,8 @@ type ChainConfig struct {
 	CortinaBlockTimestamp *uint64 `json:"cortinaBlockTimestamp,omitempty"`
 	// DUpgrade activates the Shanghai upgrade from Ethereum. (nil = no fork, 0 = already activated)
 	DUpgradeBlockTimestamp *uint64 `json:"dUpgradeBlockTimestamp,omitempty"`
+	// Cancun activates the Cancun upgrade from Ethereum. (nil = no fork, 0 = already activated)
+	CancunTime *uint64 `json:"cancunTime,omitempty"`
```

Comment:
I've added the EIP-4844 signer changes under this fork, we can change it.
However, I think we can also have a conversation around aligning forks with upstream for easier maintenance.